digraph "unitGraph" {
    "r4 := @parameter0: java.lang.String[]"
    "r5 = \"a\u7f51a\""
    "r0 = virtualinvoke r5.<java.lang.String: byte[] getBytes(java.lang.String)>(\"UTF-8\")"
    "$r1 = new java.lang.String"
    "specialinvoke $r1.<java.lang.String: void <init>(byte[],java.lang.String)>(r0, \"UTF-8\")"
    "r6 = $r1"
    "$r2 = <java.lang.System: java.io.PrintStream out>"
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.String)>(r6)"
    "goto [?= return]"
    "$r3 := @caughtexception"
    "r7 = $r3"
    "return"
    "r4 := @parameter0: java.lang.String[]"->"r5 = \"a\u7f51a\"";
    "r4 := @parameter0: java.lang.String[]"->"$r3 := @caughtexception";
    "r5 = \"a\u7f51a\""->"r0 = virtualinvoke r5.<java.lang.String: byte[] getBytes(java.lang.String)>(\"UTF-8\")";
    "r5 = \"a\u7f51a\""->"$r3 := @caughtexception";
    "r0 = virtualinvoke r5.<java.lang.String: byte[] getBytes(java.lang.String)>(\"UTF-8\")"->"$r1 = new java.lang.String";
    "r0 = virtualinvoke r5.<java.lang.String: byte[] getBytes(java.lang.String)>(\"UTF-8\")"->"$r3 := @caughtexception";
    "$r1 = new java.lang.String"->"specialinvoke $r1.<java.lang.String: void <init>(byte[],java.lang.String)>(r0, \"UTF-8\")";
    "$r1 = new java.lang.String"->"$r3 := @caughtexception";
    "specialinvoke $r1.<java.lang.String: void <init>(byte[],java.lang.String)>(r0, \"UTF-8\")"->"r6 = $r1";
    "specialinvoke $r1.<java.lang.String: void <init>(byte[],java.lang.String)>(r0, \"UTF-8\")"->"$r3 := @caughtexception";
    "r6 = $r1"->"$r2 = <java.lang.System: java.io.PrintStream out>";
    "r6 = $r1"->"$r3 := @caughtexception";
    "$r2 = <java.lang.System: java.io.PrintStream out>"->"virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.String)>(r6)";
    "$r2 = <java.lang.System: java.io.PrintStream out>"->"$r3 := @caughtexception";
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.String)>(r6)"->"goto [?= return]";
    "virtualinvoke $r2.<java.io.PrintStream: void println(java.lang.String)>(r6)"->"$r3 := @caughtexception";
    "goto [?= return]"->"return";
    "$r3 := @caughtexception"->"r7 = $r3";
    "r7 = $r3"->"return";
}
